### DISCUSSION TITLE
port power on/off via lua script

### DISCUSSION THREAD

**Discussion by f2knpw** [EdgeTX/edgetx#4004] [Q&A] [answered]
Hi,
I am quite new to EdgeTx and own a TXs16.
I have connected an ESP32 to the AUX1 serial port and successfully set a french "remoteId" beacon with GPS telemetry coming from the plane.
Currently the AUX1 port can output 5V (or not) whith the powerPort switch.

I would like to automate this pin (with some logical or physical switches of the radio). Does it exist a lua command to do this programmatically .

![portpower](https://github.com/EdgeTX/edgetx/assets/31324055/848e4914-7628-4728-a8b1-f27a1c29b2cd)

Thanks a lot for this wonderful EDGEtx !

**Comment by raphaelcoeffic** [comment, marked as answer]
I believe this is what you're after: https://luadoc.edgetx.org/part_iii_-_opentx_lua_api_reference/general-functions-less-than-greater-than-luadoc-begin-general/serialsetpower-port_nr-value?q=

**Reply by f2knpw** [reply]
Wahou thanks so much. It's exactly what I needed !
